<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modelo2</class>
 <widget class="QWidget" name="modelo2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>355</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>508</width>
    <height>355</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>508</width>
    <height>355</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="Titulo">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Elastomer version&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_20">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="MasaCub_Label">
             <property name="text">
              <string>Box Mass</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="MasaCub_Value">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>0</height>
              </size>
             </property>
             <property name="suffix">
              <string> g</string>
             </property>
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="maximum">
              <double>9999.000000000000000</double>
             </property>
             <property name="value">
              <double>55.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QLabel" name="OCub_Label">
             <property name="text">
              <string>Box Offset</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="OCub_Value">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>0</height>
              </size>
             </property>
             <property name="suffix">
              <string> mm</string>
             </property>
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="maximum">
              <double>9999.000000000000000</double>
             </property>
             <property name="value">
              <double>11.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_19">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="MasaCil_Label">
             <property name="text">
              <string>Cylinder Mass</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="MasaCil_Value">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>0</height>
              </size>
             </property>
             <property name="suffix">
              <string> g</string>
             </property>
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="maximum">
              <double>9999.000000000000000</double>
             </property>
             <property name="value">
              <double>10.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="OCil_Label">
             <property name="text">
              <string>Cylinder Offset</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="OCil_Value">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>0</height>
              </size>
             </property>
             <property name="suffix">
              <string> mm</string>
             </property>
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="maximum">
              <double>9999.000000000000000</double>
             </property>
             <property name="value">
              <double>22.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_18">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>251</width>
           <height>140</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>251</width>
           <height>140</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../../MelosimulaRF.qrc">:/images/Resources/modelo2Dimensiones.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="W_Label">
               <property name="text">
                <string> W</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="W_Value">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="suffix">
                <string> mm</string>
               </property>
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="maximum">
                <double>9999.000000000000000</double>
               </property>
               <property name="value">
                <double>32.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="H_Label">
               <property name="text">
                <string> H</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="H_Value">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="suffix">
                <string> mm</string>
               </property>
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="maximum">
                <double>9999.000000000000000</double>
               </property>
               <property name="value">
                <double>50.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="L_Label">
               <property name="text">
                <string> L</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="L_Value">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="suffix">
                <string> mm</string>
               </property>
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="maximum">
                <double>9999.000000000000000</double>
               </property>
               <property name="value">
                <double>40.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <widget class="QLabel" name="Hc_Label">
               <property name="text">
                <string>Hc</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="Hc_Value">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="suffix">
                <string> mm</string>
               </property>
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="maximum">
                <double>9999.000000000000000</double>
               </property>
               <property name="value">
                <double>35.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_16">
             <item>
              <widget class="QLabel" name="d1_Label">
               <property name="text">
                <string>d1</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="d1_Value">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="suffix">
                <string> mm</string>
               </property>
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="maximum">
                <double>9999.000000000000000</double>
               </property>
               <property name="value">
                <double>35.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_17">
             <item>
              <widget class="QLabel" name="d2_Label">
               <property name="text">
                <string>d2</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="d2_Value">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="suffix">
                <string> mm</string>
               </property>
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="value">
                <double>75.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="E_Label">
               <property name="text">
                <string> e</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="E_Value">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="suffix">
                <string> mm</string>
               </property>
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="maximum">
                <double>9999.000000000000000</double>
               </property>
               <property name="value">
                <double>11.500000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="D_Label">
               <property name="text">
                <string> d</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="D_Value">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="suffix">
                <string> mm</string>
               </property>
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="maximum">
                <double>9999.000000000000000</double>
               </property>
               <property name="value">
                <double>67.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../MelosimulaRF.qrc"/>
 </resources>
 <connections/>
</ui>
